Preprocessing

MohamedMostafa259 opened the pull request:
This branch contains a notebook for preprocessing that cleans and encodes the training dataset to be ready for making interactive dashboards and building ML models.

MohamedMostafa259 committed:
Uploaded train & test datasets

MohamedMostafa259 committed:
added dataset description in the readme file

MohamedMostafa259 committed:
uploaded notebook for cleaning and encoding the training dataset

Ahmed-M-Fayad commented:
Hey Mohammad,

Thanks for handling the data preprocessing! I'll review it and get back to you with feedback soon. Please wait until I finalize the revision.

MohamedMostafa259 committed:
Separated the `preprocessing` notebook into two notebooks `dataProcessingForML` and `dataCleaning` for organization purposes.

Updated markdown documentation.

MohamedMostafa259 committed:
Edited the name of saved datasets:
- From `train_preprocessed_plusFeatures` To `train_cleaned_plusFeatures`
- From `train_preprocessed_plusFeatures_encoded` To `train_cleaned_plusFeatures_encoded`

Ahmed-M-Fayad committed:
Thanks, Mohammad! 🚀 Your amazing work has been reviewed, and everything is good and accepted. It's now added to the GitHub repo. Great job! 👏

Ahmed-M-Fayad commented:
Thanks, Mohammad! 🚀 Your amazing work has been reviewed, and everything is good and accepted. It's now added to the GitHub repo. Great job! 👏
